src/main/java/com/hotels/mutantswarm/MutantSwarmRule.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (C) 2018-2019 Expedia, Inc.
+ * Copyright (C) 2018-2020 Expedia, Inc.
```

Comment:
2021 ?

Reply:
I think when I run `mvn package` it only updates the ones where I've made changes in 2021, and since in most of the files were edited in 2020, it doesn't update them